code/src/index.css

```diff
@@ -1,13 +1,45 @@
+/** @format */
+
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
+
+html {
```

Comment:
It is impressive that you could achieve this with so few lines of css code, nice job!🎉